src/components/Calculator.js

```diff
@@ -1,61 +1,56 @@
-          <button type="button" onClick={this.handleClick}>9</button>
-          <button className="sign" type="button" onClick={this.handleClick}>x</button>
-        </div>
-        <div className="row">
-          <button type="button" onClick={this.handleClick}>4</button>
-          <button type="button" onClick={this.handleClick}>5</button>
-          <button type="button" onClick={this.handleClick}>6</button>
-          <button className="sign" type="button" onClick={this.handleClick}>-</button>
-        </div>
-        <div className="row">
-          <button type="button" onClick={this.handleClick}>1</button>
-          <button type="button" onClick={this.handleClick}>2</button>
-          <button type="button" onClick={this.handleClick}>3</button>
-          <button className="sign" type="button" onClick={this.handleClick}>+</button>
-        </div>
-        <div className="row">
-          <button type="button" onClick={this.handleClick}>0</button>
-          <button className="equal sign" type="button" onClick={this.handleClick}>=</button>
-        </div>
+    setState((state) => calculate(state, num));
+  };
+  const { next, total } = state;
+  const displayHandler = (e) => {
+    const val = e.target.textContent;
+    setState((state) => ({ ...state, total: val }));
+  };
+  return (
+    <div className="calculator">
+      <div className="output" onChange={displayHandler}>
+        {next}
+        {total}
+        {0}
+      </div>
+      <div className="row">
+        <button type="button" onClick={handleClick}>AC</button>
+        <button type="button" onClick={handleClick}>+/-</button>
+        <button type="button" onClick={handleClick}>%</button>
+        <button className="sign" type="button" onClick={handleClick}>÷</button>
+      </div>
+      <div className="row">
+        <button type="button" onClick={handleClick}>7</button>
+        <button type="button" onClick={handleClick}>8</button>
+        <button type="button" onClick={handleClick}>9</button>
+        <button className="sign" type="button" onClick={handleClick}>x</button>
+      </div>
+      <div className="row">
+        <button type="button" onClick={handleClick}>4</button>
+        <button type="button" onClick={handleClick}>5</button>
+        <button type="button" onClick={handleClick}>6</button>
+        <button className="sign" type="button" onClick={handleClick}>-</button>
+      </div>
+      <div className="row">
+        <button type="button" onClick={handleClick}>1</button>
+        <button type="button" onClick={handleClick}>2</button>
+        <button type="button" onClick={handleClick}>3</button>
+        <button className="sign" type="button" onClick={handleClick}>+</button>
+      </div>
+      <div className="row">
+        <button type="button" onClick={handleClick}>0</button>
+        <button className="equal sign" type="button" onClick={handleClick}>=</button>
       </div>
```

Comment:
- [ ] Your calculator is not working as expected. It is impossible to perform the operations as expected. For example, if I press button `2` it gives me `20`. Also, your button modulus is not working as expected. The modulus of `20` gives `320 `which is not correct. Please Fix it :+1: